web-app/src/screens/Console/Buckets/BucketDetails/DeleteAccessRule.tsx

```diff
@@ -36,24 +37,32 @@ const DeleteAccessRule = ({
   toDelete,
 }: IDeleteAccessRule) => {
   const dispatch = useAppDispatch();
-  const onDelSuccess = () => onClose();
-  const onDelError = (err: ErrorResponseHandler) =>
-    dispatch(setErrorSnackMessage(err));
 
-  const [deleteLoading, invokeDeleteApi] = useApi(onDelSuccess, onDelError);
+  const [loadingDeleteAccessRule, setLoadingDeleteAccessRule] =
+    useState<boolean>(false);
 
   const onConfirmDelete = () => {
-    invokeDeleteApi("DELETE", `/api/v1/bucket/${bucket}/access-rules`, {
-      prefix: toDelete,
-    });
+    setLoadingDeleteAccessRule(true);
+    let wrapper: PrefixWrapper = { prefix: toDelete };
+    api.bucket
+      .deleteAccessRuleWithBucket(bucket, wrapper)
+      .then((_) => {
```

Comment:
Very minor comment, but we can remove the underscore here:

```suggestion
      .then(() => {
```